lighthouse-core/lib/page-functions.js

```diff
@@ -303,6 +303,25 @@ function getNodeLabel(node) {
   return tagName;
 }
 
+/**
+ * @param {HTMLElement} element
+ * @param {LH.Artifacts.Rect}
+ */
+function getBoundingClientRect(element) {
+  const rect = element.getBoundingClientRect();
+  // if (rect.width > 0 && rect.height > 0) {
```

Comment:
maybe do this here? do we care about never having zero-size rects?

Reply:
seems like we always want to return a rect, I'd just delete and we can filter 0 area rects later